source/index.html.md

```diff
@@ -797,12 +799,12 @@ Filter | Description
 `{ "$dateUpdated": { ">": "2020-12-31" } }` | Less than (short form) - The date is before December 31, 2020
 `{ "$dateUpdated": { "$lessThanOrEqual": "2020-12-31" } }` | The date is before or equal to December 31, 2020
 `{ "$dateUpdated": { ">=": "2020-12-31" } }` | Less than or equal to (short form) - The date is before or equal to December 31, 2020
-`{ "$dateUpdated": { "$greaterThan": "2016-04-01" } }` | The date is after April 1, 2016
-`{ "$dateUpdated": { "<": "2016-04-01" } }` | Greater than (short form) - The date is after April 1, 2016
-`{ "$dateUpdated": { "$greaterThanOrEqual": "2016-04-01" } }` | The date is after or equal to April 1, 2016
-`{ "$dateUpdated": { "<=": "2016-04-01" } }` | Greater than or equal to (short form) - The date is after or equal to April 1, 2016
+`{ "$datePublished": { "$greaterThan": "2016-04-01" } }` | The date is after April 1, 2016
+`{ "$datePublished": { "<": "2016-04-01" } }` | Greater than (short form) - The date is after April 1, 2016
```

Comment:
Just noticed this now, but the greater than and less than signs are backwards in the date filtering section.